core/src/com/etheller/warsmash/viewer5/handlers/w3x/simulation/CWorldCollision.java

```diff
@@ -154,12 +154,9 @@ public boolean intersectsAnythingOtherThan(final Rectangle newPossibleRectangle,
 						this.anyUnitExceptTwoIntersector.reset(sourceUnitToIgnore, sourceSecondUnitToIgnore))) {
 					return true;
 				}
-				if (this.groundUnitCollision.intersect(newPossibleRectangle,
-						this.anyUnitExceptTwoIntersector.reset(sourceUnitToIgnore, sourceSecondUnitToIgnore))) {
-					return true;
-				}
-				return false;
-			case FLOAT:
+				return this.groundUnitCollision.intersect(newPossibleRectangle,
+						this.anyUnitExceptTwoIntersector.reset(sourceUnitToIgnore, sourceSecondUnitToIgnore));
+				case FLOAT:
```

Comment:
Again it appears that the automatic format indentation here for `case FLOAT:` does not match all other indentation of all other case statements in this file. When we are going far out of our way to be pedantic, I think we should update cases like this to match the code around them.

Reply:
Fixed the formatting, added some more small changes in 57b2b049b8def4e26bc147360369f674e814802d.